src/category_theory/bicategory/strict.lean

```diff
@@ -63,4 +63,20 @@ instance strict_bicategory.category [bicategory.strict B] : category B :=
   comp_id' := λ a b, bicategory.strict.comp_id,
   assoc' := λ a b c d, bicategory.strict.assoc }
 
+namespace bicategory
+
+variables {B}
+
+@[simp]
+lemma whisker_left_eq_to_hom {a b c : B} (f : a ⟶ b) {g h : b ⟶ c} (η : g = h) :
+  f ◁ eq_to_hom η = eq_to_hom (congr_arg2 (≫) rfl η) :=
+by { cases η, simp only [whisker_left_id, eq_to_hom_refl] }
+
+@[simp]
+lemma whisker_right_eq_to_hom {a b c : B} {f g : a ⟶ b} (η : f = g) (h : b ⟶ c) :
```

Comment:
Thank you for you comment. I also realized that it would be better to rename `whisker_right_comp` and `whisker_right_id` in the definition of `bicategory` to `comp_whisker_right` and `id_whisker_right`, respectively, as well (probably in another PR).